app/bundles/SmsBundle/Controller/AjaxController.php

```diff
@@ -60,4 +64,37 @@ protected function getSmsCountStatsAction(Request $request)
 
         return new JsonResponse($data);
     }
+
+    /**
+     * Just selected get tokens from email  builder.
+     *
+     * @param string|null $query
+     *
+     * @return array<string,array<int|string>>
+     */
+    protected function getBuilderTokens($query): array
```

Comment:
@kuzmany the unit tests look good! Please create also a functional test that will be calling this new endpoint so we can be sure it won't ever be broken.

Reply:
@kuzmany can you please address this conversation so we can merge this PR? the code is great and can be merged